jekyll/_ccie/proxy.md

```diff
@@ -7,43 +7,32 @@ order: 7
 description: "How to get CircleCI Enterprise to use a HTTP proxy."
 ---
 
-## Proxy Support
+This document describes how to configure CircleCI to use an HTTP proxy in the following sections:
 
+* TOC
+{:toc}
 
-### Getting Started
+## Overview
 
-Lets assume that we have an unauthenticated http proxy at `10.0.0.33:3128` and the services box is at `10.0.1.238`. We use `ghe.example.com` as our github enterprise host.
+If you are setting up your proxy through Amazon, read this before proceeding: http://docs.aws.amazon.com/cli/latest/userguide/cli-http-proxy.html. Avoid proxying internal requests, especially for the Services machine. Run `export NO_PROXY=<services_box_ip>` to add it to the `NO_PROXY` rules. In an ideal case, traffic to S3 will not be proxied, and instead be bypassed by adding `s3.amazonaws.com,*.s3.amazonaws.com` to the `NO_PROXY` rule.
 
-In an ideal case, traffic to S3 will not be proxied, and instead be added to the NO_PROXY rules (to bypass it out add `s3.amazonaws.com,*.s3.amazonaws.com`  to `NO_PROXY`)
+These instructions assume an unauthenticated HTTP proxy at `10.0.0.33:3128`, a Services machine at `10.0.1.238` and use of `ghe.example.com` as the github enterprise host.
 
-The proxy instructions must be done **before** installing CircleCI on fresh vm's/instances.
+**Note:** The following proxy instructions must be completed **before** installing CircleCI on fresh VMs or instances. You must also configure JVM OPTs again as well as described below.
 
+## Service Machine Proxy Configuration
 
-### EC2 Proxies
-If you are setting up your proxy through Amazon, please read this before proceeding: http://docs.aws.amazon.com/cli/latest/userguide/cli-http-proxy.html
+The Service machine has many components that need to make network calls, as follows:
 
-You should also avoid proxying internal requests, especially for the services box. Run `export NO_PROXY=<services_box_ip>` to add it to the `NO_PROXY` rules.
-
-*Note that you must configure JVM OPTs again as well. This will be covered below*
-
-### Service Box configuration
-
-The service box has many components that need to make network calls
-
-- External Network Calls
-  - Replicated
-    - Replicated is a vendor service that we use to bootstrap CircleCI. We require them to make an outside call to validate the license. 
-    - Replicated also goes and downloads docker and installs in on the local machine. They then use a Docker container to create and configure S3 buckets.
+- **External Network Calls** - Replicated is a vendor service that we use for the Management Console of CircleCI. CircleCI  requires Replicated to make an outside call to validate the license. Replicated also downloads docker, installs it on the local machine, and uses a Docker container to create and configure S3 buckets.
```

Comment:
after "validate the license" we should add "and check for and download upgrades."